docs/conf.py

```diff
@@ -57,11 +59,13 @@
 # source_encoding = 'utf-8-sig'
 
 # The master toctree document.
-master_doc = "index"
+root_doc = "index"
 
 # General information about the project.
 project = "python-gitlab"
-copyright = "2013-2018, Gauvain Pocentek, Mika Mäenpää"
+copyright = (
+    f"2013-2018, Gauvain Pocentek, Mika Mäenpää.\n2018-{year}, python-gitlab team"
```

Comment:
Cool, adding the python-gitlab-team!

I saw an issue from Gauvain about doing this.

Reply:
Hehe yeah I think we already did that somwhere, maybe setup.py. I was thinking in a follow-up I could clean this up a bit more.